elitzur-core/src/main/scala/com/spotify/elitzur/validators/ValidationStatus.scala

```diff
@@ -39,7 +40,7 @@ private[this] case class PostValidationWrapper[A](inner: A) extends PostValidati
   override def toString: String = inner.toString
 }
 
-trait ValidationStatus[+A] extends TraversableOnce[A] {
+trait ValidationStatus[+A] extends IterableOnce[A] {
```

Comment:
scala.collection.compat._ converts this back to a TraversableOnce in 2.12.  This is why we remove the explicit overrides below: most are actually overrides for TraversableOnce.  Otherwise it won't compile in 2.13.

Reply:
Are there any alternatives to using this? Don't remember the specifics of 2.13 collections since it's been a while since I looked, sorry. I'm just mildly paranoid about having to pull in a compat lib.

Reply:
The best alternative I have found is to have separate implementations for 2.13 and 2.12 which you can do like this: https://github.com/spotify/scio/tree/master/scio-core/src/main/scala-2.12.

I think this is probably preferable though